pywhy_graphs/algorithms/pag.py

```diff
@@ -7,7 +7,12 @@
 import numpy as np
 
 from pywhy_graphs import ADMG, CPDAG, PAG, StationaryTimeSeriesPAG
-from pywhy_graphs.algorithms.generic import single_source_shortest_mixed_path
+from pywhy_graphs.algorithms.generic import (
+    has_adc,
+    inducing_path,
+    single_source_shortest_mixed_path,
+    valid_mag,
```

Comment:
Can you add these to the API list in the documentation?

Reply:
Sure.